src/test/java/seedu/address/logic/parser/DeleteCommandParserTest.java

```diff
@@ -1,32 +1,80 @@
 package seedu.address.logic.parser;
 
-/*import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
-//import static seedu.address.logic.parser.CommandParserTestUtil.assertParseFailure;
-//import static seedu.address.logic.parser.CommandParserTestUtil.assertParseSuccess;
-import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_PERSON;
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.commands.CommandTestUtil.ITEM_TYPE_INTERNSHIP;
+import static seedu.address.logic.commands.CommandTestUtil.ITEM_TYPE_INVALID;
+import static seedu.address.logic.commands.CommandTestUtil.ITEM_TYPE_NOTE;
+import static seedu.address.logic.commands.CommandTestUtil.ITEM_TYPE_PROJECT;
+import static seedu.address.logic.commands.CommandTestUtil.ITEM_TYPE_RESUME;
+import static seedu.address.logic.commands.CommandTestUtil.ITEM_TYPE_SKILL;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseSuccess;
 
 import org.junit.jupiter.api.Test;
 
-import seedu.address.logic.commands.delete.DeleteCommand;*/
+import seedu.address.logic.commands.delete.DeleteCommand;
+import seedu.address.logic.commands.delete.DeleteInternshipCommand;
+import seedu.address.logic.commands.delete.DeleteNoteCommand;
+import seedu.address.logic.commands.delete.DeleteProjectCommand;
+import seedu.address.logic.commands.delete.DeleteResumeCommand;
+import seedu.address.logic.commands.delete.DeleteSkillCommand;
+import seedu.address.model.item.Item;
+import seedu.address.testutil.TypicalIndexes;
 
-/**
- * As we are only doing white-box testing, our test cases do not cover path variations
- * outside of the DeleteCommand code. For example, inputs "1" and "1 abc" take the
- * same path through the DeleteCommand, and therefore we test only one of them.
- * The path variation for those two cases occur inside the ParserUtil, and
- * therefore should be covered by the ParserUtilTest.
- */
 public class DeleteCommandParserTest {
-//
-//    private DeleteCommandParser parser = new DeleteCommandParser();
-//
-//    @Test
-//    public void parse_validArgs_returnsDeleteCommand() {
-//        assertParseSuccess(parser, "1", new DeleteCommand(INDEX_FIRST_PERSON));
-//    }
-//
-//    @Test
-//    public void parse_invalidArgs_throwsParseException() {
-//        assertParseFailure(parser, "a", String.format(MESSAGE_INVALID_COMMAND_FORMAT, DeleteCommand.MESSAGE_USAGE));
-//    }
+
+    private DeleteCommandParser parser = new DeleteCommandParser();
+
+    @Test
+    public void parse_deleteInternship_success() {
+        assertParseSuccess(parser, "3" + ITEM_TYPE_INTERNSHIP,
+                new DeleteInternshipCommand(TypicalIndexes.INDEX_THIRD_ITEM));
+    }
+
+    @Test
+    public void parse_viewProject_success() {
```

Comment:
Lady, don't make ctrl-c ctrl-v so obvious horrr hahaha, change view -> delete, the entirety of this file?